static/js/tools/choropleth/choropleth.ts

```diff
@@ -123,34 +141,34 @@ function mouseLeave() {
  */
 function handleMapClick(geo) {
   if (geo.properties.hasSublevel) {
-    redirectToGeo(geo.properties.geoId)
+    redirectToGeo(geo.properties.geoId);
   } else {
     //TODO(iancostello): Improve this feature (change cursor)
-    alert("This geo has no further sublevels!")
+    alert("This geo has no further sublevels!");
   }
 }
 
 /**
  * Builds a redirect link given the fields to include from search params.
- * @param fields_to_include 
+ * @param fields_to_include
  * @param from_api whether the url should be to the api or locally.
  */
 function build_choropleth_url(fields_to_include, from_api) {
```

Comment:
the abstraction needed is probably to just build the search string, and to either use it for the api or a page refresh.